app/backend/prepdocslib/filestrategy.py

```diff
@@ -22,11 +22,11 @@ async def parse_file(
     if processor is None:
         logger.info("Skipping '%s', no parser found.", file.filename())
         return []
-    logger.info("Ingesting '%s'", file.filename())
+    logger.debug("Ingesting '%s'", file.filename())
```

Comment:
Looks like you changed the levels to debug, does the current output feel too verbose? I find it helpful.